### PR TITLE
Code examples for episode 209 & 210

### PR DESCRIPTION
**Pull request by int3l**

### CODE REVIEW

File: sample_code/ep210/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+Today I talk about a vulnerability I found in github actions involving the `pull_request_target` feature and how it escalates to credential access / full repository access.  I found over ~350 vulnerable repositories including ones owned by google, amazon, microsoft, alibabi, psf and more and document my experience (or lack thereof) with bug bounty programs.
+
+## Interactive examples
+
+### Bash
+
+```bash
+git clone git@github.com:<username>/gha-test
+# git clone https://github.com/<username>/gha-test
```
Comment (int3l) [inline sample_code/ep210/README.md:11]: This repo (in your `asottile` account) doesn't exist anymore, so I used a placeholder.

Comment (asottile) [reply sample_code/ep210/README.md:11]: yeah I did it on another account: https://github.com/throwaway-gha/gha-test

Comment (int3l) [reply sample_code/ep210/README.md:11]: Yes, but this is the target repository in your video. And the command itself was for cloning a fork of the target repository - which is used to create the PRs.
If you want, I can substitute the placeholder with the `throwaway-gha` username?

**Review by asottile — APPROVED** [review]

**Pull request merged**